cmd/link.go

```diff
@@ -14,22 +16,21 @@ var (
 )
 
 var linkCmd = &cobra.Command{
-	Use:   "link <target component> --component [source component]",
-	Short: "Link target component to source component",
-	Long: `Link target component to source component
+	Use:   "link <service> --component [component]",
+	Short: "Link component to a service",
+	Long: `Link component to a service
 
 If source component is not provided, the link is created to the current active
 component.
 
-In the linking process, the environment variables containing the connection
-information from target component are injected into the source component and
-printed to STDOUT.
+During the linking process, the secret that is created during the service creation (odo service create),
+is injected into the component.
 `,
-	Example: `  # Link current component to a component 'mariadb'
-  odo link mariadb
+	Example: `  # Link the current component to the 'my-postgresql' service
+  odo link my-postgresql
 
-  # Link 'mariadb' component to 'nodejs' component
-  odo link mariadb --component nodejs
+  # Link  component 'nodejs' to the 'my-postgresql' service
```

Comment:
this implementation will only link service to component, right ? 
what about previous logic, linking one component to another component.

Reply:
@surajnarwade It was decided [here](https://github.com/redhat-developer/odo/issues/652#issuecomment-429813904) that we should drop the component to component link.
If you feel this should not happen, feel free to raise a red flag :)

Reply:
agree to the point :)

Reply:
Cool :)